process/smartContract/process.go

```diff
@@ -576,25 +576,49 @@ func (sc *scProcessor) computeTotalConsumedFeeAndDevRwd(
 				sentGas, _ = core.SafeAddUint64(sentGas, outTransfer.GasLocked)
 			}
 			consumedGas, err = core.SafeSubUint64(consumedGas, sentGas)
-			log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "outTransfer.GasLimit")
+			log.LogIfError(err,
+				"function", "computeTotalConsumedFeeAndDevRwd",
+				"consumedGas", consumedGas,
+				"sentGas", sentGas,
+			)
 			consumedGas, err = core.SafeAddUint64(consumedGas, outAcc.GasUsed)
-			log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "outAcc.GasUsed")
+			log.LogIfError(err,
+				"function", "computeTotalConsumedFeeAndDevRwd",
+				"consumedGas", consumedGas,
+				"outAcc.GasUsed", outAcc.GasUsed,
+			)
 			accumulatedGasUsedForOtherShard, err = core.SafeAddUint64(accumulatedGasUsedForOtherShard, outAcc.GasUsed)
-			log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "accumulatedGasUsedForOtherShard", "outAcc.GasUsed")
+			log.LogIfError(err,
+				"function", "computeTotalConsumedFeeAndDevRwd",
+				"accumulatedGasUsedForOtherShard", accumulatedGasUsedForOtherShard,
+				"outAcc.GasUsed", outAcc.GasUsed,
+			)
 		}
 	}
 
 	moveBalanceGasLimit := sc.economicsFee.ComputeGasLimit(tx)
 	if !isSmartContractResult(tx) {
 		consumedGas, err = core.SafeSubUint64(consumedGas, moveBalanceGasLimit)
-		log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "computeGasLimit")
+		log.LogIfError(err,
+			"function", "computeTotalConsumedFeeAndDevRwd",
+			"consumedGas", consumedGas,
+			"moveBalanceGasLimit", moveBalanceGasLimit,
+		)
 	}
 
 	consumedGasWithoutBuiltin, err := core.SafeSubUint64(consumedGas, accumulatedGasUsedForOtherShard)
-	log.LogIfError(err, "computeTotalConsumedFeeAndDevRwd", "accumulatedGasUsedForOtherShard")
+	log.LogIfError(err,
+		"function", "computeTotalConsumedFeeAndDevRwd",
+		"consumedGas", consumedGas,
+		"accumulatedGasUsedForOtherShard", accumulatedGasUsedForOtherShard,
```

Comment:
can you also print the result?

Reply:
done

Reply:
The result is always 0.

Reply:
Correct, but in case we might change the default value returned in SafeSubUint64 and SafeAddUint64 in case of an error, this might prove useful